projects/addon-editor/components/editor-new/editor-new.component.ts

```diff
@@ -117,7 +116,12 @@ export class TuiEditorNewComponent
     }
 
     onMouseDown(event: MouseEvent) {
-        if (this.editor?.isFocused) {
```

Comment:
Could you clarify this change?

Reply:
Yes, it is fix for issue with focus. Previously, focus was lost when clicking on a disabled button in a toolbar or wrapper padding.

Reply:
Maybe preventing default on mousedown would be preferable? On the items that you don't want to drag the focus away.

Reply:
I set preventing mousedown on toolbar and move padding to prosemirror container. This is much better, thanks!